blogs/posts/2024-02-11_sankey_plot.qmd

```diff
@@ -0,0 +1,565 @@
+---
+title: "Visualising participant recruitment with Sankey plots"
+author:
+  - Craig Parylo
+date: "2024-02-11"
+categories: [learning, tutorial, visualisation]
+reference-location: margin
+citation-location: margin
+---
+
+```{r}
+#| code-fold: true
+#| warning: false
+#| message: false
+#| output: false
+
+# libraries
+library(tidyverse)       # 'tidy' data wrangling
+library(plotly)          # sankey visualisation framework
+library(reactable)       # viewing interactive datatables
+library(glue)            # concatenating strings
+library(tidygraph)       # api for graph / network manipulation
+library(scales)          # used for colour transformation
+```
+
+# Introduction
+
+Sankey diagrams are great tools to visualise flows through a system. They show the flow / conversion rate after each step of a process and the width of the arrows is proportional to the flow / conversion rate.
+
+I'm working on a project to evaluate the effectiveness of a risk screening programme for people aged between 55-74 years and a history of smoking[^1]. One report requirement is to be able to visualise how people have engaged with the programme, from recruitment, attendance at appointments, their outcome from risk assessment, low-dose CT scans and will eventually cover the impact of the screening on early detection of those diagnosed with lung cancer.
+
+[^1]: For more background on the TLHC programme see <https://www.england.nhs.uk/contact-us/privacy-notice/how-we-use-your-information/our-services/evaluation-of-the-targeted-lung-health-check-programme/>
```

Comment:
I reckon this footnote should be a sentence or paragraph in the main body of the article so it's clear what the work was for and to provide some more context.

Reply:
Thanks, I've added on overview of the programme to the main body and kept the URL as a footnote for further information.